### PR TITLE
added login methods

### PR DESCRIPTION
**Pull request by MoSilvera**

### CODE REVIEW

**Review by croninsw — DISMISSED** [review]
I think this looks good, and I'm happy to approve but first I need tree fiddy.

File: src/scripts/registration.js [hunk 1]
```diff
@@ -39,8 +39,32 @@ const registrationHandler = {
...
+        return `<form id="loginForm">
+        <fieldset>
+            <label for="username">Username:</label>
+            <input type="text" name="loginUsername" id="loginUsername"></input>
+        </fieldset>
+        <fieldset>
+            <label for="password">Password:</label>
+            <input type="password" name="loginPassword" id="loginPassword"></input>
+        </fieldset>
+        <fieldset>
```
Comment (naienko) [inline src/scripts/registration.js:53]: where my login button?

**Review by naienko — APPROVED** [review]
:100:

**Review by croninsw — APPROVED** [review]
:+

**Pull request merged**